Helpers/time.js

```diff
@@ -0,0 +1,24 @@
+export function convertTime(state) {
+  let newTime = state.replace(/T.*/, "")
+
+  newTime = newTime.slice(2, newTime.length).split("-").reverse().join("-")
+
+  return newTime
+}
+
+export function completedInDays(createdAt) {
+  let todoAddTime = createdAt.replace(/Created At: /, "")
+
+  todoAddTime = todoAddTime.split("-").reverse().join("-")
+
+  let todoFinishTime = new Date().toISOString().replace(/T.*/, "")
```

Comment:
We use `const` for the variables that does not change over time
```suggestion
  const todoFinishTime = new Date().toISOString().replace(/T.*/, "")
```